nuxt.config.ts

```diff
@@ -149,7 +150,7 @@ export default defineNuxtConfig({
       },
       posthogDisabled: '',
       appUrl: '',
-      googleMapsApiKey: '',
+      googleMapsApiKey: process.env.GOOGLE_MAPS_API_KEY_PUBLIC,
```

Comment:
Google maps key already provided here in public namespace, you can use it also in server side. Also no change is necessary here.